### PR TITLE
De-duplicated and re-arranged basic rules

### PR DESCRIPTION
**Pull request by bleehu**
added Primary and Secondary Stats to glossary
removed some feats rules from feats document, added feats section to basic rules
re-wrote secondary stats section
fixed b0rk'd headings not diplaying properly

More work is yet to be done; in particular, the character creation section is still duplicated. But I figured this already had enough scope creep to get bogged, so I'll continue in another PR.
	modified:   01_01_Basic_Rules.txt
	modified:   03_Glossary_of_Terms.txt
	modified:   CharacterCreation/03_Feats.txt

### CODE REVIEW

File: 01_01_Basic_Rules.txt [hunk 1]
```diff
@@ -1,12 +1,20 @@
...
 ==============================
 Premise
 ==============================
 
-	Welcome to Compound X! The specific time and place that your adventure takes
-place is up to your Game Master (often called a GM for short), but all of them
-take place far in the future where humans have left earth on their faster-than-light
-spaceships to explore dangerous new worlds fraught with friend and foe of all
-manner of robots and aliens! 
+	The specific time and place that your adventure takes place is up to your
```
Comment (rosebrallier) [inline 01_01_Basic_Rules.txt]: lang (sc)- I'd swap this to "time and place of your adventure is up to..." b/c it reads less awk

Comment (rosebrallier) [reply 01_01_Basic_Rules.txt]: s-v agreement: "adventures"

File: 01_01_Basic_Rules.txt [hunk 2]
```diff
@@ -1,12 +1,20 @@
...
 
-	Welcome to Compound X! The specific time and place that your adventure takes
-place is up to your Game Master (often called a GM for short), but all of them
-take place far in the future where humans have left earth on their faster-than-light
-spaceships to explore dangerous new worlds fraught with friend and foe of all
-manner of robots and aliens! 
+	The specific time and place that your adventure takes place is up to your 
+Game Master (often called a GM for short), but all of them take place far in the 
+future where humans have left earth on their faster-than-light spaceships to 
+explore dangerous new worlds fraught with friend and foe of all manner of robots
```
Comment (rosebrallier) [inline 01_01_Basic_Rules.txt]: "friends and foes"

File: 01_01_Basic_Rules.txt [hunk 3]
```diff
@@ -118,12 +126,22 @@ see the 00_New_Player_Walkthrough.txt document in CharacterCreation. Following
...
 
+Secondary Stats:
+	These stats are derived from your primary stats and determine things like
+how fast your character moves and how quickly they learn new skills. When making
+a new character, you'll calculate these after picking Primary Stats.
+
+Stat and Combat Modifiers:
+	These numbers are calculated from your stats, and are often added to dice
+rolls to show how your character's physical and mental abilities affect how they
+do at different tasks.
```
Comment (rosebrallier) [inline 01_01_Basic_Rules.txt]: lang- "how well they perform different tasks"

File: 01_01_Basic_Rules.txt [hunk 4]
```diff
@@ -346,23 +293,76 @@ table, below.
...
+	Your Character's Secondary stats are each equal to one of your Primary 
+Stats, but no single Primary Stat can be used for two of your Secondary Stats.
 
 Movement Speed:
-	Your speed in meters, for every two actions. Use (Movement Speed)/2 for one 
-action (drop the remainder). For more information, see topic "Movement and 
-Sprinting", below.
+	Your Movement Speed is how fast your character can move every two Actions in
+combat, measured in meters. Use (Movement Speed)/2 for one Action and drop the 
+remainder. For more detail on movement in combat, see the "Movement and
```
Comment (rosebrallier) [inline 01_01_Basic_Rules.txt:304]: New Issue(?): Oops. I haven't been dropping the remainder before multiplying when moving for two actions.
Honestly I like my way better.

File: 01_01_Basic_Rules.txt [hunk 5]
```diff
@@ -346,23 +293,76 @@ table, below.
...
+every level. Every time you level up, you gain skill points equal to double your 
+"Skill Point Gain". Your Skill Point Gain should be equal to either your 
+Intelligence or your Charisma.
+
+== Secondary Stat Assignment Examples ==
+	Charisma and Intelligence are associated with Skill Point Gain. If you have 
+5 Charisma and 6 Intelligence, you would have 6 Skill Point Gain.
+
+	Let's now assume you have 6 Strength, 3 Dexterity, and 4 Fortitude. You would 
+have 3 Movement Speed (from your Dexterity) and 4 Carry Ability (from your Fortitude).
```
Comment (rosebrallier) [inline 01_01_Basic_Rules.txt]: *"could have"

File: 01_01_Basic_Rules.txt [hunk 6]
```diff
@@ -377,8 +377,9 @@ Some class feats and other abilities can increase the number of Nanites you can
 hold and use at a given time. Also, Unlike Health, Nanites regenerate during 
 combat at a rate of 3 Nanites per turn. 
 	
-
-===== Assigning Stats to a New Character =====
+==============================
+Creating a New Character
+==============================
 
 	When making a new character, you'll have to assign their Base stats.
```
Comment (rosebrallier) [inline 01_01_Basic_Rules.txt]: *"Primary Stats"

File: 01_01_Basic_Rules.txt [hunk 7]
```diff
@@ -391,7 +392,7 @@ level 15. After level 15, the limit for stat points is 20. The number of
 initial base stat points that you have to apply is listed with your Race or
```
Comment (rosebrallier) [inline 01_01_Basic_Rules.txt:393]: "primary stat points"

File: 01_01_Basic_Rules.txt [hunk 8]
```diff
@@ -438,21 +439,86 @@ increase per level.
 Saving Throws
 ==============================
 
-	Saving throws are used in dire or critical situations to defend against very 
+	Saving Throws are used in dire or critical situations to defend against very 
 negative effects.
 
 - Will: defend against a mind affect (Ex: throw off hopelessness after crashing in an escape pod)
 - Shock: maintain composure after serious damage (Ex: stay conscious after being shot by a high caliber round)
 - Reflex: quickly react to an event (Ex: dodge falling debris, fire at an enemy if they leap from one piece of cover to another)
```
Comment (rosebrallier) [inline 01_01_Basic_Rules.txt]: "react quickly" (reason- parallel structure)

File: 01_01_Basic_Rules.txt [hunk 9]
```diff
@@ -438,21 +439,86 @@ increase per level.
...
 ==============================
 
-	Saving throws are used in dire or critical situations to defend against very 
+	Saving Throws are used in dire or critical situations to defend against very 
 negative effects.
 
 - Will: defend against a mind affect (Ex: throw off hopelessness after crashing in an escape pod)
 - Shock: maintain composure after serious damage (Ex: stay conscious after being shot by a high caliber round)
 - Reflex: quickly react to an event (Ex: dodge falling debris, fire at an enemy if they leap from one piece of cover to another)
 - Awareness: knowledge and understanding of surroundings (Ex: realizing someone has walked up behind you)
```
Comment (rosebrallier) [inline 01_01_Basic_Rules.txt]: Breaks the parallel structure altogether :(

File: 01_01_Basic_Rules.txt [hunk 10]
```diff
@@ -438,21 +439,86 @@ increase per level.
...
+Feats
+==============================
+
+	As your character levels up and gets better at what they do, they develop
+certain talents and abilities that help them differentiate themselves from their
+peers. These special tallents and abilities are called "Feats". See the 
+"Leveling Up Rules" table to see how many feats you get at which levels. Unless 
+otherwise specified, you cannot take a feat more than once.
+
+	There are Five kinds of feats and we'll detail them below;
```
Comment (rosebrallier) [inline 01_01_Basic_Rules.txt]: *five; "feats, detailed below" (reason: maintains 2nd person)

File: 01_01_Basic_Rules.txt [hunk 11]
```diff
@@ -438,21 +439,86 @@ increase per level.
...
+
+* Passive Feats
+* Active Feats
+* Weapon Expertise Feats
+* Stat Feats
+* Class Feats
+
+Passive Feats:
+
+	When you take a feat, you may take a passive feat. Passive Feats are used
```
Comment (rosebrallier) [inline 01_01_Basic_Rules.txt:478]: I'd nix the first sentence of Passive, Active and Weapon feats and make a note in leveling up about which feats can be chosen when. Or specify "When you gain a non-Class feat upon leveling up..."

File: 01_01_Basic_Rules.txt [hunk 12]
```diff
@@ -438,21 +439,86 @@ increase per level.
...
+	When you take a feat, you may decide to take a Weapon Expertise Feat. Weapon
+Expertise Feats are a special kind of passive feat, that get better as you 
+increase your skill with a particular kind of weapon group. 
+
+	These feats represent dedicated training with a particular class of weapon, 
+and offer bonuses based on the relevant Weapons skill (Weapon skills are listed 
+in "06_Skills.txt"). Each feat offers a set of bonuses related to using that type of 
+weapon (and only that type of weapon), and each bonus requires the character to have 
+a number of points in the relevant Weapon skill. A character with 0 points in 
+Weapons - Pistols would gain only one benefit, while 60 points would benefit from
```
Comment (rosebrallier) [inline 01_01_Basic_Rules.txt]: "while a character with 60 points would..."

File: 01_01_Basic_Rules.txt [hunk 13]
```diff
@@ -438,21 +439,86 @@ increase per level.
...
+in "06_Skills.txt"). Each feat offers a set of bonuses related to using that type of 
+weapon (and only that type of weapon), and each bonus requires the character to have 
+a number of points in the relevant Weapon skill. A character with 0 points in 
+Weapons - Pistols would gain only one benefit, while 60 points would benefit from 
+them all. The feats are formatted as a list of "X: Benefit", where X is the number of 
+points required in the corresponding Weapon skill to gain the benefit.
+	
+	Weapon Expertise feats require proficiency with the weapon type, but otherwise 
+have no prerequisites. Note that as all characters are proficient with Unarmed 
+Strike, Melee Expertise effectively has no prerequisite.
```
Comment (rosebrallier) [inline 01_01_Basic_Rules.txt]: @Turtlelord26 Does it make sense that Unarmed is the pre-req for Melee? I'd think that Unarmed Expertise should be its own thing, and the pre-req for Melee should be proficiency with a melee weapon type.

File: 01_01_Basic_Rules.txt [hunk 14]
```diff
@@ -438,21 +439,86 @@ increase per level.
...
+
+Primary Stat Feats:
+	
+	Stat Feats are special and cannot be taken when you get a feat like Active
+or Passive Feats. Instead, they are immediately unlocked when you fulfill their
```
Comment (rosebrallier) [inline 01_01_Basic_Rules.txt]: "like Active, Passive or Weapon Expertise feats"

File: 01_01_Basic_Rules.txt [hunk 15]
```diff
@@ -438,21 +439,86 @@ increase per level.
...
+Luck or Charisma requirement.
```
Comment (rosebrallier) [inline 01_01_Basic_Rules.txt]: In the interest of making space for future developments how about "Stat requirement" as opposed to "Luck or Charisma"?

**Review by rosebrallier — APPROVED** [review]
I threw in a bunch of wording suggestions for funsies.
Thank you for all the necessary but less thrilling work here @bleehu !

**Comment by bleehu** [comment]
I cannot tell you how much I want to just merge this and move on. But If you're going to take the time to do a full review of the document, the least I can do is make the corrections. Will execute and then merge.

**Comment by bleehu** [comment]
@rosebrallier, made the changes. Want me to wait for another review, or just shipit?

**Comment by rosebrallier** [comment]
Shipit!!!!!! (So many the thanks!)

Rose Brallier

File: 01_01_Basic_Rules.txt [hunk 16]
```diff
@@ -444,8 +445,8 @@ negative effects.
 
 - Will: defend against a mind affect (Ex: throw off hopelessness after crashing in an escape pod)
 - Shock: maintain composure after serious damage (Ex: stay conscious after being shot by a high caliber round)
-- Reflex: quickly react to an event (Ex: dodge falling debris, fire at an enemy if they leap from one piece of cover to another)
-- Awareness: knowledge and understanding of surroundings (Ex: realizing someone has walked up behind you)
+- Reflex: React quickly to an event (Ex: dodge falling debris, fire at an enemy if they leap from one piece of cover to another)
+- Awareness: Keep apprised of your surroundings (Ex: realizing someone has walked up behind you)
```
Comment (rosebrallier) [inline 01_01_Basic_Rules.txt:449]: This is a fantastic solution!!!

File: 01_01_Basic_Rules.txt [hunk 17]
```diff
@@ -497,19 +498,20 @@ and offer bonuses based on the relevant Weapons skill (Weapon skills are listed
...
 Strike, Melee Expertise effectively has no prerequisite.
 
 Primary Stat Feats:
 	
-	Stat Feats are special and cannot be taken when you get a feat like Active
-or Passive Feats. Instead, they are immediately unlocked when you fulfill their
-Luck or Charisma requirement. 
+	Stat Feats are special and cannot be taken when you get a feat like Active,
+Passive, or Weapon Expertise Feats. Instead, they are immediately unlocked when 
+you fulfill their stat requirement.
```
Comment (rosebrallier) [inline 01_01_Basic_Rules.txt]: I love the Oxford comma!

**Review by rosebrallier — APPROVED** [review]

File: 01_01_Basic_Rules.txt (hunk 11 above)
Comment (Turtlelord26) [reply 01_01_Basic_Rules.txt:478]: Second the removal of the first sentences.

File: 01_01_Basic_Rules.txt (hunk 4 above)
Comment (Turtlelord26) [reply 01_01_Basic_Rules.txt:304]: Fortunately, the rules do not mandate dropping any remainders when moving 2 actions. The base unit is Move Speed per 2 Actions, and the division only happens when moving for exactly 1 Action.
You have been correct

File: 01_01_Basic_Rules.txt [hunk 18]
```diff
@@ -438,21 +440,87 @@ increase per level.
...
+weapon (and only that type of weapon), and each bonus requires the character to have 
+a number of points in the relevant Weapon skill. A character with 0 points in 
+Weapons - Pistols would gain only one benefit, while a character with 60 points 
+would benefit from them all. The feats are formatted as a list of "X: Benefit", 
+where X is the number of points required in the corresponding Weapon skill to 
+gain the benefit.
+	
+	Weapon Expertise feats require proficiency with the weapon type, but otherwise 
+have no prerequisites. Note that as all characters are proficient with Unarmed 
+Strike, Melee Expertise effectively has no prerequisite.
```
Comment (Turtlelord26) [inline 01_01_Basic_Rules.txt:508]: Nice to see these in this document 🐢

**Review by Turtlelord26 — APPROVED** [review]
Very, very shiny!

Aside from two remaining nb inlines from Rose, very shipit the thing!

**Pull request merged**